pyproject.toml

```diff
@@ -44,7 +44,7 @@ packages = [
 ]
 
 [tool.poetry-dynamic-versioning]
-enable = true
+enable = false
```

Comment:
I know your feelings about the dynamic versioning plugin, but uh... Probably just an oversight I'm sure 😁

Reply:
I think it is good! Just a lil rascally sometimes

Reply:
https://github.com/linkml/linkml/pull/2208/commits/7fed49b3764d3f8c1ee84422444e675d0baec99c